Fix for a problem with decryption of first frame in Webkit.

skywojciechowskim opened the pull request:
Summary: Fix for a problem with decryption of first frame in Webkit.
Type: Fix
Test Plan: Unittests, Component Tests, YT Conformance Tests
Jira: RIALTO-499

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/github-triggered/52206/ started

rdkcmf-jenkins commented:
https://jenkins.code.rdkcentral.com/job/build-component/91540/ started.

rdkcmf-jenkins commented:
https://jenkinsrdkm.cmf.code.rdkcentral.com/job/github-triggered/16110/ started

rdkcmf-jenkins commented:
https://jenkinsrdkm.cmf.code.rdkcentral.com/job/build-component-platform/23846/ started.

skywojciechowskim commented:
> Coverage statistics of your commit: WARNING: Lines coverage decreased from: 85.3% to 85.2% Functions coverage stays unchanged and is: 92.2%

Coverage report failed, because I removed some code. Changed GstGenericPlayer code is fully covered with UTs

rdkcmf-jenkins commented:
Sanity tests for ['RPI4 Mediaclient'] added.
https://jenkinsrdkm.cmf.code.rdkcentral.com/job/test-components-flow/44891/

rdkcmf-jenkins commented:
Sanity tests for ['BCM Reference Refapp2'] added.
https://jenkinsrdkm.cmf.code.rdkcentral.com/job/test-components-flow/44894/
